pkg/modules/organization/usecase.go

```diff
@@ -0,0 +1,15 @@
+package organization
+
+import (
+	"context"
+
+	"github.com/SigNoz/signoz/pkg/types"
+	"github.com/SigNoz/signoz/pkg/valuer"
+)
+
+type Usecase interface {
```

Comment:
```suggestion
type Usecase interface {
```
Let's start calling this `Module`. `Usecase` seems like a forced introduction of a new concept.